src/main/java/org/mastodon/mamut/feature/branch/BranchNSpotsFeature.java

```diff
@@ -46,13 +46,136 @@
 import org.mastodon.properties.IntPropertyMap;
 import org.scijava.plugin.Plugin;
 
+/**
+ * Represents the number of spots within a branch spot. In many cases this may
+ * be equivalent to {@link BranchDisplacementDurationFeature} (duration).
+ * However, in situations where there are spots missing within a branch spot, this
+ * may well be different.
+ *
+ * <h1>Spot-Graph</h1>
```

Comment:
The "Spot-Graph" is usually called "ModelGraph" in the code? I would suggest to stay with the "ModelGraph" term. Similarly I would suggest to use the term "Branch-Graph" rather than "BranchSpot-Graph".